src/sass/base/_global.sass

```
@@ -0,0 +1,18 @@
+@import "../base/variables"
+
+html
+    font-size: 13px
+
+    body 
+        font-family: $primary-font
+        font-weight: $regular
```

```suggestion
        font-weight: $fw-regular
```
Để các tên biến dựa tên viết tắt cho rõ ràng nhé
VD: font-weight -> fw, font-size -> fs